docs/source/content/user_documentation/basyx_components/v2/submodel_repository/features/registry-integration.md

```diff
@@ -0,0 +1,12 @@
+# Registry Integration
+This feature automatically integrates the Descriptor with the Registry while creation of the Submodel at the Repository. <br>
+It also automatically removes the Descriptor from the Registry when the Submodel is removed from the Repository.
+
+To enable this feature, the following two properties should be configured:
+
+```
+basyx.submodelrepository.feature.registryintegration = {Submodel-Registry-Base-Url}
+basyx.externalurl = {Submodel-Repo-Base-Url}
+```
+
+This feature gets enable automatically when both of the above defined properties are configured, i.e., no external enabled/disabled property is required.
```

Comment:
> This feature gets enable automatically when both of the above defined properties are configured, i.e., no external enabled/disabled property is required.

&rarr; > This feature **_is_** enabled ~automatically~ when both of the above  defined properties are **_set_**. I.e., no **_additional_** enabled/disabled property is required.